crates/sui-indexer/src/handlers/committer.rs

```diff
@@ -29,7 +31,7 @@ pub async fn start_tx_checkpoint_commit_task<S>(
 
     info!("Indexer checkpoint commit task started...");
     let checkpoint_commit_batch_size = std::env::var("CHECKPOINT_COMMIT_BATCH_SIZE")
-        .unwrap_or(5.to_string())
+        .unwrap_or(CHECKPOINT_COMMIT_BATCH_SIZE.to_string())
```

Comment:
Why did we change this number as well?

Reply:
these configs are ones we used in deployment config overriding, these changes are to push them to codes for the simplicity of deployment configs.